05_classify_mutations_multimodal/compare_mlp.ipynb

```diff
@@ -1,5 +1,14 @@
 {
```

Comment:
<p>It never ceases to amaze me how much bigger model variance is than the effect of different model types </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Right! We've seen this a lot with the mutation prediction problem (and probably lots of other problems are this way too, especially in biology).</p>